### DISCUSSION TITLE
failed to resolve file

### DISCUSSION THREAD

**Discussion by guyzilberblum** [Cyfrin/security-and-auditing-full-course-s23#183] [Q&A]
Hello im trying to do an audit in code 4 arena and i did forge test and this is what  i got 
"""

    failed to resolve file: "/root/2024-05-arbitrum-foundation/node_modules/@openzeppelin/contracts- 
       upgradeable/proxy/utils/Initializable.sol": No such file or directory (os error 2); check configured remappings
        --> /root/2024-05-arbitrum-foundation/src/bridge/AbsBridge.sol
        @openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol
 
"""

help plzzzzzz  by the way its a clone code

**Comment by EngrPips** [comment]
```js
failed to resolve file: "/root/2024-05-arbitrum-foundation/node_modules/@openzeppelin/contracts- 
   upgradeable/proxy/utils/Initializable.sol": No such file or directory (os error 2); check configured remappings
    --> /root/2024-05-arbitrum-foundation/src/bridge/AbsBridge.sol
    @openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol
```

The above seems like you haven't built the project yet, and some dependencies are missing. You should look on the contest page; there should be information on how to successfully build the repo. Better still, ask on the Discord channel of the contest on Code4rena, and a fellow contestant will help you with building the repo.

**Comment by guyzilberblum** [comment]
Ok, I will check.

**Reply by EngrPips** [reply]
Yeah, I am convinced you would get more help on code4rena regarding a contest you are doing on the platform than anywhere else.

**Comment by PatrickAlphaC** [comment]
Love how you're asking questions! But let's keep this forum for questions about the curriculum, thanks!
